.bazelrc

```diff
@@ -1,6 +1,9 @@
 # import common bazelrc shared with e2e workspaces
 import %workspace%/.bazelrc.common
 
+# Mock versioning command to test the --stamp behavior
+build --workspace_status_command="echo BUILD_SCM_VERSION 1.2.3"
```

Comment:
Is there any other way to do this? Having test data within the root bazelrc isn't the greatest imo :/

We could do `build:stamp-test` or something like that, but that means the tests need to do `--config=stamp-test` which might be too much of a hassle? Can it go in a bazelrc in a subfolder? IDK if there's any solution worth doing at this time 🤷

Reply:
Moved to `.bazelrc.testing` to keep things clean